### ISSUE TITLE
Syntax error in main.py from Source v0.7.2.4.tar.gv

### ISSUE THREAD

**Issue by bscheek** [Amulet-Team/Amulet-Map-Editor#124]
I used the instructions from [here](https://github.com/Amulet-Team/Amulet-Map-Editor) and the source tar.gz. from [here](https://github.com/Amulet-Team/Amulet-Map-Editor/releases/tag/v0.7.2.4) (v0.7.2.4.tar.gz.) to install.

When using `python main.py` to launch, an error was thrown, but no traceback was shown due to a syntax error on line 13 of main.py.  The problem is an errant "f" at the beginning of the line.  Removing it got the traceback working again so error messages would show.

**Comment by Podshot** [comment]
Hmm, that’s odd, what version of Python do you have installed?

**Comment by gentlegiantJGC** [comment]
The f before the string is valid python. It is Python 3's way of doing string formatting

**Comment by gentlegiantJGC** [comment]
Most likely it isn't using python 3

**Comment by bscheek** [comment]
I installed the default Python distro for Ubuntu 20.04 LTS which included Python 2.7 and 3.8, but which marks 2.7 as default.  I installed PIP in 3.8 so I used the python3.8 command to install.  And then after having lots of dependency issues I realized that I needed to set 3.8 as default.  So likely your right that at the point of the syntax error 2.7 was still default.  I’ll try adding the “f” back now and see if this is resolved.
